src/SOS/SOS.UnitTests/Scripts/WebApp.script

```diff
@@ -116,17 +116,3 @@ VERIFY:\s+MT\s+Count\s+TotalSize\s+Class Name\s+
 VERIFY:\s*<HEXVAL>\s+<DECVAL>\s+<DECVAL>\s+.*
 VERIFY:\s*Total\s+<DECVAL>\s+objects\s+
 !VERIFY:.*UNKNOWN.*
-
-SOSCOMMAND:DumpAsync
-VERIFY:\s*Statistics:\s+
-VERIFY:\s+MT\s+Count\s+TotalSize\s+Class Name\s+
-VERIFY:\s*<HEXVAL>\s+<DECVAL>\s+<DECVAL>\s+.*
-VERIFY:\s*Total\s+<DECVAL>\s+objects\s+
-
-SOSCOMMAND:DumpAsync -mt <POUT>\s+MT\s+Count\s+TotalSize\s+Class Name\s+(<HEXVAL>)\s+<DECVAL>\s+<DECVAL>\s+.*<POUT>
-VERIFY:\s*Statistics:\s+
-VERIFY:\s+MT\s+Count\s+TotalSize\s+Class Name\s+
-VERIFY:\s*<HEXVAL>\s+<DECVAL>\s+<DECVAL>\s+.*
```

Comment:
It's just missing a (<< Awaiting:)?  here to pass the test

Reply:
How do I run them locally?

Reply:
from the `SOS.UnitTests`  dir run `dotnet test -c debug--logger "trx;logfilename=testResults.trx" -l "console;verbosity=detailed" --filter "DisplayName=SOS.WebApp3" -d`

Reply:
Thanks, but that command is corrupted (fails with errors like:
```
Required argument missing for option: -d
MSBUILD : error MSB1009: Project file does not exist.
Switch: trx;logfilename=testResults.trx
```
and when I try to fix it, I get a whole bunch of execution errors about dump files not being found, pipes not being available, etc.  What should the command be?

Reply:
I just ran `dotnet test -c debug --logger "trx;logfilename=testResults.trx" -l "console;verbosity=detailed" --filter "DisplayName=SOS.WebApp3"` (looks like you can drop the -d and I missed a space between debug and --logger). I get: 

![image](https://user-images.githubusercontent.com/19413848/162511204-3953277f-61d2-4b12-9ef0-03d89e3b840f.png)

Can you push the test changes and I will add on it?

Reply:
> Can you push the test changes and I will add on it?

Thanks, I'd welcome the assistance.